### ISSUE TITLE
When adding an attribute group to product, only one attribute is added at a time

### ISSUE THREAD

**Issue by exeqtwasnotfree** [atrocore/atrocore#974]
If I what to add an attribute group with 3st attributes in, I only get one added to product.
If I try again I get one more.
And if I add a third time I get the last one.
The point of adding a group fails, if I only get one at a time.

**Comment by AnnZast** [comment]
This can happen if all three attributes are in different attribute tabs. If they are in the same tab, they should be added at the same time.

**Comment by exeqtwasnotfree** [comment]
They are in the same tab and in same group.
You can´t add attributes or groups from other attribute-tabs, only the ones that belong to the tab is selectable.

**Comment by AnnZast** [comment]
Yes, but it is possible that attribute group contains attributes from different tabs. In this case only those attributes that belong to the current tab will be added.

I tried to reproduce it on demo and everything is correct. When I added attribute group Engraving to the product, three attributes were linked at the same time https://demo-extended.atropim.com/#Product/view/icecat_9932269

**Comment by exeqtwasnotfree** [comment]
But you don´t have attribut-tabs in demo, I wonder if that have something to do with it?

**Comment by exeqtwasnotfree** [comment]
I have made an attribute tab : https://demo-extended.atropim.com/#AttributeTab/view/b657318cbc08229f8
And some attributes from group Processor are in this tab (6 of them).
I made user "user_for_both_editor" (pass 1234) member of the role editor and made some adjustments to its rights.
If I deny this role to create attibute-tabs, the user can no longer add group processor in tab Test.
In your demo this adds all six of the attributes, but in my site it only adds one at a time.
